TROUBLESHOOTING.md

```diff
@@ -0,0 +1,420 @@
+# Troubleshooting
+
+## Terminology
+
+* **seed project**: An existing GCP project with resources, services, and service accounts needed to create projects with the project factory.
+* **service account**: The service account (which exists in the **seed project**) that creates the **target project**.
+* **target project**: The project being generated by the Project Factory.
+
+- - -
+## Problems
+
+* [Seed project missing APIs](#seed-project-missing-apis) - The seed project is missing required APIs.
+* [Service account missing roles](#service-account-missing-roles) - The service account has insufficient permissions.
+* [Common issues](#common-issues)
+
+### Seed project missing APIs
+
+The Project Factory requires the following services are enabled on the seed project. If these APIs are not enabled they can cause the Project Factory to get into a bad state.
```

Comment:
Let's include a link to the README section which provides the exhaustive list, in case this gets out of sync.

https://github.com/terraform-google-modules/terraform-google-project-factory/tree/adrienthebo/troubleshooting-guide#apis

Also let's add links from the README back to this FAQ for each of the APIs (I liked that you included an explanation of why the API is needed.